File: src/QuickApp/ClientApp/src/app/models/report.ts
export interface IReport {
    id: number;
    shortDescription: string;
    priority: number;
    fullDescription: Date;
    clientId: string;
    status: number;
}

export class Report implements IReport {
    id: number;
    shortDescription: string;
    priority: number;
    fullDescription: Date;
    clientId: string;
    status: number;
}

File: src/DAL/Models/TFS/Report.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace FSW_TFS.DAL.Models.TFS
{
    public class Report
    {
        [Key]
        public int Id { get; set; }
        public string ShortDescription { get; set; }
        public int Priority { get; set; }
        public string FullDescription { get; set; }
        public string ClientId { get; set; }
        public int Status { get; set; }
    }
}